certified-connectors/Progressus Advanced Projects/apiProperties.json

```diff
@@ -0,0 +1,36 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "username": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Username",
+          "description": "The Username for this api",
+          "tooltip": "Provide the Username",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": true,
+            "required": "true"
+          }
+        }
+      },
+      "password": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Web Service Access Key",
+          "description": "The Web Service Access Key for this api",
+          "tooltip": "Provide the Web Service Access Key",
+          "constraints": {
+            "tabIndex": 3,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#ffffff",
```

Comment:
@ProgressusGlobal The apiProperties.json should have a valid brand color and doesn't use an invalid brand color, #007ee5 or #ffffff. Please provide a valid one.